tests/resampler.rs

```diff
@@ -21,4 +18,56 @@ mod tests {
         assert_eq!((expected_in_rate, expected_out_rate), st.get_rate());
     }
 
+    #[test]
+    fn test_process_float() {
+        let mut st = State::new(1, RATE, RATE, 4).unwrap();
+        assert_eq!((1, 1), st.process_float(42, &[8.5], &mut [9.5]).unwrap());
+    }
+
+    #[test]
+    fn test_skip_zeros() {
```

Comment:
like we discussed, I don't really know what I'm doing here, but they may sense, so we may it leave it for now.

same for `test_latency`

Reply:
@tmatth can you have a look?